packages/@romefrontend/css-parser/index.test.ts

```diff
@@ -0,0 +1,30 @@
+import {parseCSS} from "@romefrontend/css-parser";
+import {createFixtureTests} from "@romefrontend/test-helpers";
+import {removeCarriageReturn} from "@romefrontend/string-utils";
+
+const promise = createFixtureTests(async (fixture, t) => {
+	const {files} = fixture;
+
+	const inputFile = files.get("input.css");
+	if (inputFile === undefined) {
+		throw new Error(`The fixture ${fixture.dir} did not have an input.css`);
+	}
+
+	const filename = inputFile.relative;
+
+	const inputContent = removeCarriageReturn(inputFile.content.toString());
+
+	const ast = parseCSS({
+		input: inputContent,
+		path: filename,
+	});
+
+	const outputFile = inputFile.absolute.getParent().append(
+		inputFile.absolute.getExtensionlessBasename(),
+	).join();
+
+	t.snapshot(ast, undefined, {filename: outputFile});
+});
+
+// @ts-ignore Doesn't support top-level await
+await promise;
```

Comment:
As long as this is declared as a file handler with a `parse` method you can replace this with:

```suggestion
import {declareParserTests} from "@romefrontend/test-helpers";

const promise = declareParserTests();

// @ts-ignore Doesn't support top-level await lol
await promise;
```